### ISSUE TITLE
Duplicate error can be annoying

### ISSUE THREAD

**Issue by trebmuh** [mvo5/synaptic#56]
When one have a duplicate in its source.list / source.list.d/*, then synaptic is issuing a window pop up message like this one:
![synaptic-error](https://user-images.githubusercontent.com/8705846/92119569-498ef780-edf8-11ea-8839-ff9abd32974a.png)

It is scarring for the user since it's written "Error" while it should only be a warning since it doesn't prevent Synaptic from running normally, it just slows it down because it pings different addresses. Suggestion is that it should be a "Warning" pop up windows instead of a "Error" one.

Going further, it'd be awesome to have a tick-box option somewhere to avoid synaptic to show this window if I choose so.

**Comment by mvo5** [comment]
Yeah, I agree that this error is not great. It requires a bit of work to fix this, apt is giving us this data but does not provide much more so it's hard for synaptic to know if it's a real scary error (like a corrupted cache) or just something harmless like this.

**Comment by trebmuh** [comment]
Thanks for your answer @mvo5 .

Going beyond than further:

> ...it'd be awesome to have a tick-box option somewhere to avoid synaptic to show this window if I choose so.

As a live-build maker, I'd like to have an option for this "error" message not to be shown. I'm not sure how to do so, just throwing the idea here.
